### PR TITLE
[MRG] Add new pricing to our website

### PR DESCRIPTION
**Pull request by fursinger**
We change our pricing page to follow a more common way of presenting price plans. It will be easier to understand and contain detailed information about the differences between our plans.

### CODE REVIEW

**Comment by fursinger** [comment]
Example for a pricing card. The `more` can be dropped. Instead of `price`, `currency` and `period`, you could also `use no-price-text`.

```
  {{< pricing-card
    price=12
    currency=CHF
    period=Monat
    small="bei jährlicher Abrechnung (CHF 144.-/Jahr), bei monatlicher Abrechnung: CHF 16.-/Monat"
    feature="60 Signaturen pro Jahr inklusive"
    feature-small="Zusätzliche Signaturen werden verrechnet mit: CHF 1.20 (EES), CHF 1.70 (FES), CHF 2.50 (QES)."
    cta-label="Jetzt loslegen"
    cta-href=""
    cta-target=""
    more="Mehr erfahren"
  >}}
  {{% markdown %}}
  ## Personal
  Für Einzelpersonen, 
  die effiziente Arbeit schätzen.
  {{% /markdown %}}
  {{< /pricing-card >}}
```

**Comment by fursinger** [comment]
Screenshot_2020-09-17 Pläne und Preise

File: site/content/english/pricing.md [hunk 1]
```diff
@@ -22,11 +22,11 @@ With Skribble you sign your contracts within seconds - globally legally valid, s
...
     price=2.50
-    currency=CHF
-    period=Signatur
-    small="up to an upper limit of CHF 30, plus signatures from external parties*."
+    currency=CHF/
+    period=signature
+    small="up to a maximum of CHF 30.-, plus signatures from external parties*."
     feature="2 signatures/ month free of charge"
-    feature-small="*If you invite others to sign, a further CHF 2.00 to 2.50 per signature is added."
+    feature-small="*If you invite other people to sign, an additional amount of CHF 2.00 to 2.50 per signature is added."
```
Comment (Kirastina) [inline site/content/english/pricing.md:29]:
```suggestion
    feature-small="*If you invite other people to sign, an additional CHF 2.00 to 2.50 per signature is added."
```

File: site/content/english/pricing.md [hunk 2]
```diff
@@ -54,7 +54,7 @@ With Skribble you sign your contracts within seconds - globally legally valid, s
   {{< pricing-card
     featured=true
     price=85
-    currency=CHF
+    currency=CHF/
     period=Month
```
Comment (Kirastina) [inline site/content/english/pricing.md:58]:
```suggestion
    period=month
```

**Comment by dkumpf1** [comment]
[DEPLOY-PREVIEW]

Tested: OK

I tested this multiple times on every browser (Chrome/FF/CHromium) and also on mobile (Native/Chrome) in every language (DE/EN/FR) and we still have a bug in french ("Sur demande" on mobile doesn't look good), but decided, that this isn't a blocker.

**Pull request merged**